config/config.yml

```diff
@@ -63,6 +63,10 @@ sidebar:
       path: /to-other-languages
       title: ':bicyclist: To other languages'
 
+    - order: 8
+      path: /projects
+      title: ':package: Projects'
```

Comment:
Should we be more specific? For example, `Interesting projects by Thai developers`.
CC @dmakeroam @ponggun @surrealist What do you think?

Reply:
@aaronamm Interesting topic, but in the future I may have a page that can search all project with given tags, audient can easily access project.

for me, the "Interesting projects" is enough.

Reply:
@mildronize  This reminds me the first .NET admin meeting that we wanted to target only .NET projects made by Thai developers because we want to make .NET Thailand is very specific community and focus. 

There are a lot of open source projects out there but how we can help Thai developers to promote their projects.

.NET Thailand want to be niche and very specific, I think is good for our marketing strategy.
IMHO, I have thought something like `Interesting .NET projects by Thai developers`.